index.css

```diff
@@ -22,13 +41,28 @@ a {
   text-decoration: none;
 }
 
```

Comment:
[OPTIONAL] I advise you to make the HTML scroll behavior smooth, this will make it look more pleasing 😀.

```
html {
  scroll-behavior: smooth;
}
```